pkg/csvcopy/csvcopy.go

```diff
@@ -223,16 +226,18 @@ func (c *Copier) Copy(ctx context.Context, reader io.Reader) (Result, error) {
 	}
 
 	start := time.Now()
-	wg.Add(1)
 	go func() {
-		defer wg.Done()
```

Comment:
the reader doesn't need to be in the wait group as this task will always complete before workers finish tasks